demos/time_dependent_rayleigh_benard/timedep-rayleigh-benard.py.rst

```diff
@@ -0,0 +1,201 @@
+Rayleigh-Benard Convection
```

Comment:
The name should be different from the other RBC demo
```suggestion
Time dependent Rayleigh-Benard Convection
```

Reply:
Good catch.